### ISSUE TITLE
GEF, icon is missing: icon "palette_view.gif" (org.eclipse.gef/icons/palette_view.gif)

### ISSUE THREAD

**Issue by speckyspooky** [eclipse/gef#119]
Currently we work on the next release of BIRT 4.17.
There we use the latest eclipse plattform version:
Version: 2024-09 (4.33)
Build id: I20240815-1810

The eclipse-designer with active BIRT-view has one register with the icon of a color palette normally.
But currently the icon is not longer listed and the eclipse warning means that the icon is missing from "org.eclipse.gef/icons/palette_view.gif)" - see screen below.

Instead of the palette icon we get a red rectangle or a small fram-icon (as default icon).

Is the cion complettly removed or is the icon changed?
What can we do to fix it?

I tried a small research here on GEF but I figured out no good hint of a change for that icon.

![grafik](https://github.com/user-attachments/assets/a7540908-3394-4090-833c-19f40712cacf)

**Comment by merks** [comment]
I was looking into this as well.  My theory is as follows. In the workbench e4 serialization it records the image url of the parts, e.g. the palette view. But now gef has changed most images from.gif to .png so the url saved in the workbench is no longer usable.  I could find no reference to the old url in BIRT.  I should have tried to create a brand new workspace to see if that too has a problem. If so then probably gef as a broken url in some plugin.xml.

**Comment by azoitl** [comment]
Yes GEF  Classic replaced that .gif icon with a png version. In current GEF Classic only the png version is referenced, either in code or in the plugin.xml files. We had similar experiences in 4diac IDE when updating views or icons that the e4 model was still referring to the old icons. Is there some way except deleting the workspace's .metadata directory to fix that?

**Comment by azoitl** [comment]
BTW this is the wrong GEF for this report. The right place would be here: https://github.com/eclipse/gef-classic
Can we somehow move the issue?
